repo_server/instances.yml

```diff
@@ -0,0 +1,32 @@
+---
+instance_list:
```

Comment:
I realise state has been defaulted so isn't strictly necessary here, but could you add it in for documentation/example purposes?. Then if we need to remove these instances, its obvious where to set absent.

Reply:
Fair enough ... I've updated repo_server/instances.yml and the commented instances outline in playbooks/setup_openstack_instances.yml.